### ISSUE TITLE
Windows bindings stopped working when I upgraded to Big Sur 11.5.1

### ISSUE THREAD

**Issue by kgust** [jasonrudolph/keyboard#78]
Last night I upgraded from Big Sur 11.5 (Intel) to Big Sur 11.5.1. This morning I realized that most of my windows bindings weren't working. (I.E. `ctrl-s` + `h`)

All of the alpha bindings no longer work. The symbol bindings (`,`, `.`, `space`, `return`, etc.) still appear to be working.

I realized that my versions of Hammerspoon, Karabiner-Elements, and keyboard were all of out date. I spent a bit of time upgrading everything. When that didn't fix it, I removed my configuration and all of the apps and reinstalled from a fresh copy of keyboard.

Now with a fresh copy of everything, nothing has changed. Since I just upgraded to Big Sur 11.5.1 last night, something must have changed with macOS.

Any ideas?

**Comment by jasonrudolph** [comment]
👋 @kgust: Thanks for reporting this. I'm running Big Sur 11.5.1 as well, but I haven't seen this issue. I'm not sure what would be causing this problem. 😕  If you're able to figure it out, please let me know.

**Comment by kgust** [comment]
After not working for a week, everything is suddenly working this morning. I can't figure out why; all of the software versions have remained the same.

Big Sur **11.5.1**
Hammerspoon **0.0.90 (6030)**
Karabiner-Elements **13.5.0**

Oh, wait. There was _one_ change since I opened this issue. I reverted the `.hammerspoon` config directory to the original version, the version I had when I first encountered this problem.

I'm just grateful this is working again. I think we can close this for now. I can re-open if it occurs again.
